packages/tracing-internal/src/browser/web-vitals/onTTFB.ts

```diff
@@ -19,20 +19,20 @@ import { bindReporter } from './lib/bindReporter';
 import { getActivationStart } from './lib/getActivationStart';
 import { getNavigationEntry } from './lib/getNavigationEntry';
 import { initMetric } from './lib/initMetric';
-import type { ReportCallback, ReportOpts } from './types';
-import type { TTFBMetric } from './types/ttfb';
+import { whenActivated } from './lib/whenActivated';
+import type { MetricRatingThresholds, ReportOpts, TTFBReportCallback } from './types';
+
+/** Thresholds for TTFB. See https://web.dev/articles/ttfb#what_is_a_good_ttfb_score */
+// NOTE (Sentry): We use a lower threshold here than web-vitals, which uses [800, 1800]
```

Comment:
Note to self: I lowered this so tests would pass etc. Not sure if it makes sense to not capture this when lower than 800 🤷 we can revisit this later.